lib/omniauth/strategies/openid_connect.rb

```diff
@@ -315,7 +315,7 @@ def id_token_callback_phase
       def valid_response_type?
         return true if params.key?(options.response_type)
```

Comment:
I guess it should be changed here too.

Reply:
There's no need for that Krzysiek, `params` uses `HashWithIndifferentAccess.new` internally and it will accept both symbol and string keys

Reply:
Oh, it's `HashWithIndifferentAccess` so OK.

Reply:
But yeah, to be safe, it could be applied here too, I guess that we can't guarantee that in all use cases the params hash will have `indifferent_access`